src/core/tree.ml

```diff
@@ -459,7 +460,7 @@ module CM = struct
     let tc_fst_cmp (tca, _) (tcb, _) = TC.compare tca tcb in
     List.sort_uniq tc_fst_cmp (List.filter_map keep_treecons telst)
 
-  (** [store m c] is true iff a term of column [c] of matrix [m] contains a
+  (** [store m c] is true iff a term of column [c] of matrix [m] contains a - @PROBLEM?
```

Comment:
The signature doesn't match.

Reply:
Are you sure?

Reply:
What is [vars] array?

Reply:
Ah yes, it's a cross reference, it is the array in the `walk_tree` function